bin/process-manifests.sh

```diff
@@ -16,6 +16,10 @@ GIT_COMMIT_SENTINEL=$( cat $SENTINEL_FILENAME 2> /dev/null || true )
 BUILD_ARTIFACTS=""
 EXIT_STATUS=0
 
+echo CIRCLE_PR_USERNAME $([ -v CIRCLE_PR_USERNAME ] && echo true || echo false)
+echo CIRCLE_PR_REPONAME $([ -v CIRCLE_PR_REPONAME ] && echo true || echo false)
+echo CIRCLE_PR_NUMBER $([ -v CIRCLE_PR_NUMBER ] && echo true || echo false)
```

Comment:
@gsfr Are the above echo statements necessary, or just confirming their availability while coding? If we want to keep them for debug, why not print their value rather than true/false?